### ISSUE TITLE
[🐛 Bug]: Pages not support multiple search parameters in the URL with the same KEY

### ISSUE THREAD

**Issue by VShkotin** [cloudflare/next-on-pages#409]
### next-on-pages environment related information

**System**:
        Platform: darwin
        Arch: arm64
        Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:23 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6020
        CPU: (12) arm64 Apple M2 Pro
        Memory: 16 GB
        Shell: /bin/zsh

**Binaries**:
        Node: 18.16.1
        Yarn: 1.22.19
        npm: 9.8.1
        pnpm: N/A

**Package Manager Used**: yarn (classic)

**Relevant Packages**:
        @cloudflare/next-on-pages: 1.3.0
        vercel: N/A
        next: N/A

### Description

Can't get multiple search params on the SSR page:

For example, we have such page: 

```typescript
export const runtime = "edge";

export default async function Test({ params, searchParams }: any) {
  console.log("params", params);
  console.log("searchParams", searchParams);

  return null;
}
``` 

in case I open this URL 
```typescript
http://127.0.0.1:8788/tests/ttt?a=1&a=2&b=3
``` 

I see in the console:
<img width="476" alt="image" src="https://github.com/cloudflare/next-on-pages/assets/29312565/4c46b67c-83b8-452d-82c1-2549d7df4440">

but the search params should be like this one, with multiple "a" values 
```typescript
a: ["1", "2"],
b: "3"
``` 

### Pages Deployment Method

None

### Would you like to help?

- [ ] Would you like to help fixing this bug?

**Comment by james-elicx** [comment]
Hello, I can see if your environment info that you're using v1.3.0. Please can you try using the latest version instead - I believe we fixed this in v1.3.1.

**Comment by VShkotin** [comment]
Oh, thank you a lot @james-elicx ! yeah, in the 1.3.1 version this works correctly) I'll close this issue
